### ISSUE TITLE
Change the rocket pod acquisition range from 2000 to 1300

### ISSUE THREAD

**Issue by mjstahlberg** [Unvanquished/gameplay#67]
**Please vote this issue up :+1: or down :-1: using the thumb reactions to
denote your sentiment on the change proposed in the title!**

Issue addressed (optional): *None*
Related merge request (optional): *None*

- - -

*Polls may be started by developers to gauge community support for a
prospective or existing merge request concerning a small change; usually a
moderate update to the default value of a setting that affects the gameplay.
The change as such should be completely described in the issue title; a
rationale may be given in the comments.*

*Polls are neither necessary nor sufficient for making changes to the game.
They support, but do not replace, review and approval of the merge request.*

**Comment by mjstahlberg** [comment]
This is in fact a minor change as 2000 is essentially unbounded and rarely reached. A range of 1300 is where it starts to get interesting: An advanced dragoon may be able to barely hit the pod outside of its acquisition range, though only in an outside area where the goon can shoot in a 45° angle and where the rocket pod is not located higher than the goon. From the 1300 mark, we can re-evaluate and adjust in smaller steps if needed.

**Comment by DolceTriade** [comment]
What's the purpose of the change? Is this a nerf? If so, why?

**Comment by mjstahlberg** [comment]
> What's the purpose of the change? Is this a nerf? If so, why?

It's technically a nerf but you can also think of it as a sanity check. Maps where ranges above 1300 play a role are possibly those that suffer most under the rocket pod, balance wise. You need quite a large room or long hall for this change to take effect. For instance, it makes a small difference on Niveus:

![unvanquished_2023-04-24_221307_000](https://user-images.githubusercontent.com/3780206/234492152-b6139a67-f6d6-4907-b995-0cb44559fe1d.jpg)
(barely outside the proposed 1300 qu range; note the two blue squares)

![unvanquished_2023-04-24_221311_000](https://user-images.githubusercontent.com/3780206/234492195-ea4fcb57-e101-465f-89f7-4794b69e03a3.jpg)
(just within 1300 qu)

Note that I'm not even fully convinced that the rocket pod is overpowered since you have quite a few options to play around it, but I've heard players suggest it may be, and I am now mildly traumatized by a match on Niveus where there were four or five of them in this spot and you would die almost immediately if a human player managed to block your attempt to rush into the base with their body. I think it's agreed upon that it **can** be frustrating, and the effectively infinite range of 2000 qu (maps where this can be exceeded may exist but I don't recall any) seems to me like the ideal knob to turn to *not* nerf the pod in less frustrating scenarios.

TL;DR: This proposal sets the acquisition range to a saner default, it has effects only in corner cases but we can adjust it in smaller steps from here.

**Comment by ghost** [comment]
the exagerated range of rockets plus their exagerated agility is a real problem for alien beginners. It's also a state1 defense, which should thus not be that strong, considering the adv goon which have barbs is stage 3 IIRC.
